recipes/pyfpgrowth/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pyfpgrowth" %}
+{% set version = "1.0" %}
+{% set sha256 = "a3e5780385241d90d3ae5cabcca767eb65427b0a908014d42164b3ea8899f1d0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pyfpgrowth
+
+about:
+  home: https://github.com/evandempsey/fp-growth
+  license: BSD
```

Comment:
It looks like the license is `ISC`:

https://github.com/evandempsey/fp-growth/blob/master/LICENSE